packages/docmanager/src/manager.ts

```diff
@@ -128,7 +128,7 @@ export class DocumentManager implements IDocumentManager {
         if (!handler) {
           return;
         }
-        handler.saveInterval = value || 120;
```

Comment:
This file does not use the symbolic constant you created earlier

Reply:
@JasonWeill  I have added the symbolic constant to the constants file and imported the constant from the file. I believe this should fix the issue.